src/site/content/en/blog/codelab-signin-form/index.md

```diff
@@ -30,89 +32,134 @@ As you'll see, these elements enable built-in browser functionality, improve acc
 Add the following code inside the `<body>` element:
 
 ```html
-<form id="signin" action="#" method="post">
-
+<form id="login" action="#" method="post">
   <h1>Sign in</h1>
-
   <section>
     <label>Email</label>
     <input />
   </section>
-
   <section>
     <label>Password</label>
     <input />
   </section>
-
   <button>Sign in</button>
-
 </form>
 ```
-{% Aside 'codelab' %}
-[View complete code for Step 1](https://glitch.com/edit/#!/signin-form-codelab-1).
-{% endAside %}
+
+Here's how your `index.html` should look at this point:
+
+<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/login-form-best-practices-codelab-step-1?path=index.html&attributionHidden=true&sidebarCollapsed=true"
+    alt="How the Glitch should look after completing step 1."
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
```

Comment:
Looks like this: 

![image](https://user-images.githubusercontent.com/4713486/82515045-1be1a500-9acc-11ea-83fe-4fcbb01fb90e.png)

I think this is probably the easiest way to maintain the code samples.

Reply:
I like it — enables people not to have to click away from the article.